sw/airborne/subsystems/gps/gps_datalink.c

```diff
@@ -30,20 +30,136 @@
 #include "subsystems/gps.h"
 #include "subsystems/abi.h"
 
+#include <stdio.h> // REMOVE
+
+#ifdef GPS_USE_DATALINK_SMALL
```

Comment:
Please check such flags with `#if foo` instead of `#ifdef foo`, see also http://docs.paparazziuav.org/latest/stylec.html#stylecpp